src/patterns/addresses/error-messages/index.njk

```diff
@@ -23,7 +23,7 @@ stylesheets:
     id: "address-line-1",
     name: "addressLine1",
     errorMessage: {
-    text: "Enter address line 1"
+    text: "Enter address line 1, typically the building and street"
```

Comment:
@CharlotteDowns Added this new commit to match the error message example in Addresses pattern

Reply:
Nice! Thank you @calvin-lau-sig7